index.html 

```diff
@@ -0,0 +1,42 @@
+<! doctype html>
+<head>
+	<title> Bubble Tea site</title>
+</head>
+<body>
+	< div class=“topnav”>
+		<img src =“ ” ALT=“logo” Width=600 Height = 400”/>
+		<a class= “active” href=#home”>Home</a>
+		<a href=#news”>News</a>
+		< a href=“about”>About</a>
+		<a href=“contact”>Contact</a>
+	</div>
+	<div>
+	<img src =“”  Alt=“description of image”/>
+</div>
+
+<div> 
+	<p> text </p>
+</div>
+
+
+<div>
+	<p>text </p>
+</div>
+
+<div>
+	<img src =“”  Alt=“description of image”/>
+</div>
+
+<div>
+	<p> Text </p>
+	<button>More </button>
+
+</div>
+
+<footer>
+	<p>copyright2020</p>
+	<p>Terms of condition</p>
+	<p>Contact us</p>
+</footer>
+	</body>
```

Comment:
again, tab issues with body here...need to put this entire file through an html prettifier